### PR TITLE
SEO-3267 - JS overview http to https

### PR DESCRIPTION
**Pull request by Sharly-Atieno**
Hi @Aishwarya-Ganesan,
please review the changes for JS overview.

Regards,
Sharly A.

### CODE REVIEW

File: js/Overview.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: Welcome to Syncfusion Essential JS
+title: About Essential JavaScript Platform | Syncfusion
```
Comment (gowthamanthavasiyappan) [inline js/Overview.md:3]: Revert this.

**Review by gowthamanthavasiyappan — CHANGES_REQUESTED** [review]

**Comment by gowthamanthavasiyappan** [comment]
Commented

File: js/Overview.md (hunk 1 above)
Comment (Sharly-Atieno) [reply js/Overview.md]: done

**Review by gowthamanthavasiyappan — APPROVED** [review]

**Comment by gowthamanthavasiyappan** [comment]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**